backend/src/dataAnalyticsTest.py

```diff
@@ -0,0 +1,30 @@
+from dataAnalytics import dataAnalytics
+from playsSeperator import playsSeperator
+from dataCollector import dataCollector
+
+def test03():
+  da = dataAnalytics()
```

Comment:
Can you rename the variables so that it is easier to understand? Something like data_analytics instead of da so